Resources/Prototypes/Body/Prototypes/decoy.yml

```diff
@@ -0,0 +1,8 @@
+﻿- type: body
+  id: Decoy
+  name: decoy
+  root: root
+  # fake part
+  slots:
+    root:
+      part: null
```

Comment:
Is this for buckle coz looks sus af.

Reply:
yes its sus af
having no root part crashes client, not having body means it cant be buckled by verb or dragging

Reply:
You should fix that rather than doing dodgy workarounds.